### PR TITLE
DR3.1 Fixes

### PR DESCRIPTION
**Pull request by geordie666**
Additional fixes to the webpage for the "DR3.1" update based on comments from @weaverba137

### CODE REVIEW

File: pages/dr3/issues.rst [hunk 1]
```diff
@@ -7,65 +7,65 @@ DR3.1 Update to sweeps and Tractor Files
...
-     tractor-1763p240.fits tractor-1821p295.fits tractor-1888p270.fits tractor-1916p295.fits
-     tractor-1934p255.fits tractor-1942p130.fits tractor-1942p255.fits tractor-1942p320.fits
-     tractor-1951p237.fits tractor-1896p300.fits tractor-1985p267.fits tractor-1986p237.fits
-     tractor-1986p272.fits tractor-1965p300.fits tractor-2030p270.fits tractor-2040p287.fits 
-     tractor-2049p237.fits tractor-2054p187.fits tractor-2063p300.fits tractor-2087p305.fits
-     tractor-2087p272.fits tractor-2109p317.fits tractor-2115p317.fits tractor-2116p302.fits
-     tractor-2117p307.fits tractor-2176p097.fits tractor-2204p252.fits tractor-2224p285.fits
-     tractor-2254p290.fits tractor-2261p257.fits tractor-2290p292.fits tractor-2295p320.fits
-     tractor-2310p320.fits tractor-2318p302.fits tractor-2334p200.fits tractor-2496p257.fits
+  tractor-1763p240.fits tractor-1821p295.fits tractor-1888p270.fits tractor-1916p295.fits
```
Comment (weaverba137) [inline pages/dr3/issues.rst]: Is there a particular reason to favor subheadings over dictionary lists here?  Also, by removing the indenting in this particular spot, you've actually undone the fixed-width formatting.

File: pages/dr3/issues.rst [hunk 2]
```diff
@@ -7,65 +7,65 @@ DR3.1 Update to sweeps and Tractor Files
...
-   directory structure is the sweeps files, for which the uncorrupted versions are available at
-   legacysurvey/dr3/sweep/3.1 (instead of at legacysurvey/dr3/sweep/3.0). Note that users who
-   want access to the original version of the legacysurvey directory structure can still 
-   find it at legacysurvey/dr3.0
+These files have been reprocessed and are included in a "DR3.1" directory structure,
+which is now the default to which DR3 directories point. The one exception to the overall
+directory structure is the sweeps files, for which the uncorrupted versions are available at
+legacysurvey/dr3/sweep/3.1 (instead of at legacysurvey/dr3/sweep/3.0). Note that users who
+want access to the original version of the legacysurvey directory structure can still 
+find it at legacysurvey/dr3.0
```
Comment (weaverba137) [inline pages/dr3/issues.rst]: You didn't actually fix the punctuation here.  Look at the last sentence.

File: pages/dr3/issues.rst [hunk 3]
```diff
@@ -6,87 +6,79 @@
...
 
 Workaround
-----------
-   Which proposal IDs were considered in which brick can be obtained from the ccds files that
-   are documented on the `catalogs page`_.
+  Which proposal IDs were considered in which brick can be obtained from the ccds files that
+  are documented on the `catalogs page`_.
 
-.. _`catalogs page`: ../catalogs
+  .. _`catalogs page`: ../catalogs
```
Comment (weaverba137) [inline pages/dr3/issues.rst]: I think you have inadvertently commented out this link.

File: pages/dr3/issues.rst (hunk 1 above)
Comment (geordie666) [inline pages/dr3/issues.rst]: It looks better. However, my opinion on this isn't a strong one so I've changed it back to a list.

**Comment by weaverba137** [comment]
In reStructuredText, there shouldn't be any leading space before a `..` directive.

File: pages/dr3/issues.rst (hunk 2 above)
Comment (geordie666) [inline pages/dr3/issues.rst]: The question of whether to end a potentially ambiguous link with a period is a point of contention between technical manuals and journals, e.g., http://ux.stackexchange.com/questions/15226/should-a-sentence-ending-with-a-url-terminate-with-a-period#15228
Again, you seem to have a stronger opinion than me about this, so I added a period.

File: pages/dr3/issues.rst (hunk 3 above)
Comment (geordie666) [inline pages/dr3/issues.rst]: nikola is ambivalent to whether links are indented, but I changed it for rigor.

**Comment by weaverba137** [comment]
Please install the current master on your test site so we can take a final look before tagging.

**Comment by geordie666** [comment]
Yep, doing that now. It takes a few minutes to serve.

**Pull request merged**